commands/weather.js

```diff
@@ -0,0 +1,37 @@
+const { got } = require('./../got');
+  //usage: bb weather <city>
+module.exports = {
+  name: 'weather',
+  description: 'Shows the current weather in a given city',
+  ping: true,
+  permission: 100,
+  category: 'Info command',
+  noBanphrase: true,
+  execute: async (channel, user, input) => {
+    try {
+      if (input.length < 3) {
+        return 'Usage: bb weather <city>';
+      }
+      const city = input[2];
+      const url = `https://www.timeanddate.com/weather/${city}`;
```

Comment:
Tried with 'Nevada', 'New York'. Doesn't seem to work as it gave me a 404.

Reply:
Well, new york wouldn't work since it's two words.

Could support multiple words and uri encode it

Reply:
Well yeah, i did url encode it with %20. 

Anyways, i don't really see a reason for this command. Supibot already has the best weather command out there. 🤷

Reply:
not false